template.yaml

```diff
@@ -9,17 +9,27 @@ variants:
           psel: 0x0
     memory_map:
       - !Ram
-          range:
-            start: 0x20000000
-            end: 0x20020000
-          access:
-            boot: false
-          cores:
-            - main
+        name: ITCM
+        range:
+          start: 0x00000000
+          end: 0x00010000
+        access:
+          boot: true
+        cores:
+          - main
+      - !Ram
+        name: DTCM
+        range:
+          start: 0x20000000
+          end: 0x20020000
+        access:
+          boot: true
+        cores:
+          - main
       - !Nvm
           range:
             start: 0x90000000
-            end: 0x98000000
+            end: 0x94000000
```

Comment:
This change is due to using only one chip, right? I'd use both of them in dual mode. So, remember to revert this back, as soon as we implement dual mode.

Reply:
I would switch to dual mode as soon as we understand performance issue. Otherwise it's "premature optimization".